cluster/saltbase/salt/docker/docker-healthcheck

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is intended to be run periodically, to check the health
+# of docker.  If it detects a failure, it will restart docker using systemctl.
```

Comment:
If docker failed to restart due to some reasons after 60s waiting time, before systemctl to restart it, we have to wait for:

10s (healthcheck timer) + 30s (grace before restarting) = 40s

But in GCE salt-based configuration, the wait time is 10s (healthcheck timer) before restart it. Just want to point out the difference here.